README.md

```diff
@@ -1,13 +1,12 @@
 # minimap-cursorline package [![Build Status](https://travis-ci.org/atom-minimap/minimap-cursorline.svg?branch=master)](https://travis-ci.org/atom-minimap/minimap-cursorline)
 
-Displays Atom cursorline in the minimap.
+Highlights the current line in Atom's [minimap](https://github.com/atom-minimap/minimap).
```

Comment:
We could keep this change but it should like to the package instead of the repo.

I think "the current line" is incorrect because it highlight every line with a cursor on it when there are multiple cursors.

```suggestion
Highlights a line with a cursor in Atom's [minimap](https://atom.io/packages/minimap).
```